libcontainer/configs/config.go

```diff
@@ -205,15 +205,37 @@ type Config struct {
 type Hooks struct {
 	// Prestart commands are executed after the container namespaces are created,
 	// but before the user supplied command is executed from init.
+	// Note: This hook is now deprecated
 	Prestart []Hook
 
+	// CreateRuntime commands MUST be called as part of the create operation after
+	// the runtime environment has been created but before the pivot_root has been executed.
+	// CreateRuntime is called immediately after the deprecated Prestart hook.
+	CreateRuntime []Hook
+
```

Comment:
suggest adding TODOs for CreateContainer and StartContainer (or just one TODO to finish sync of changes to https://github.com/opencontainers/runtime-spec/blob/master/specs-go/config.go#L130)